libnmstate/__init__.py

```diff
@@ -32,6 +32,8 @@
 
 ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
 
+_MINIMUM_NM_VERSION = "1.22.8"
```

Comment:
Sorry for the confusion, this does not work because this causes a cyclic dependency. Please move this constant into the nmclient.py file.

Reply:
Thank you, I'll change that. But also getting another error because `NM.utils_version()` function will return the version ID of libnm but not the version. Can you please suggest what function I should use to get the libnm version?

Reply:
#903 contains example code to decode/encode the version, please take a look